### ISSUE TITLE
Endnotes?

### ISSUE THREAD

**Issue by vr8hub** [robinwhittleton/edgar-allan-poe_poetry#1]
I'm sure I'm missing something obvious and will regret having to ask such a stupid question, but where are the endnotes coming from? I see a _few_ of them in the scans, but, e.g. I do not see the notes for either of the "To Helen"s in the scans or in the PG transcription, and the metadata lists P.F. Collier as the only one with an "ann" role. Same for the note on "Tamerlane," and so on, judging from what I see in the scans, for most of the notes.

**Comment by vr8hub** [comment]
One more, since it's different, and I'll await a reply. The first paragraph of note-3 on "Al Aaraaf" is in the scans (and transcription), but the rest of that note, including the different version of the poem, is not. Where did the rest come from?

**Comment by robinwhittleton** [comment]
So the way the original scanned volume is set up is odd. There are footnotes, but there explanatory notes after each section too. To take the example of “Al Aaraaf”, the initial footnote is on [page 309](https://archive.org/details/worksedgarallan00unkngoog/page/n319/mode/1up), but the additional notes start on [page 361](https://archive.org/details/worksedgarallan00unkngoog/page/n371/mode/1up). I’ve tried to assign them all as endnotes to the poems where it makes sense.

To be honest, I’d forgotten about this when I wrote the request for review. This has been in production for a while 😅

**Comment by vr8hub** [comment]
No worries, I knew you didn't make them up, I just couldn't find them. :) Thanks for the pointer!
In Note 32 (in the scans), only the first few words of the first sentence is about "To Helen"; the rest of the first sentence is about other poems, and the next sentence and the missing lines are all about "Romance." It seems odd to print missing lines from "Romance" in a note about "To Helen." Wouldn't that part of the note ("The poem styled …) be better in a note on "Romance"?

You said, "I fixed it all up to meet the scans" in your email. In "Dreams," e.g., the "In climes from my imagining—apart" is from PG rather than the scans, rather than the scans "Inclines of my imaginary apart" with a footnote of "Inclines of my imaginging apart? Ed." (no em-dash). How/why did you decide to keep PG in this case rather than go with the scans?

Sorry for all the questions, I'm just trying to figure out what I need to pay attention to. Before I had footnotes in the production I couldn't find in the scans, now I have footnotes in the scans that aren't in the production. :) I'm just trying to eliminate the things I don't have to care about, but I don't know I don't have to care about them without asking about them. Given your statement above, it's theoretically possible you might have missed a change, but it's more likely you did it on purpose, I just need to make sure it's the latter.

I will say all of this would probably be a good candidate for production notes, i.e. "The transcription is from an earlier edition than the scans, the current edition was made to match the scans except in these cases for these reasons," so we know what's what and why later when no one remembers. :) Maybe even "The scans have notes in two places, most with the poems themselves and others after all of the poems," although maybe anyone but me would have found them. (A distinct possibility.)

**Comment by robinwhittleton** [comment]
On “To Helen” / “Romance”, that’s a reasonable point. I guess the question is: does the endnote make sense if split into two, given that it’s about what’s in the volumes? Maybe we should duplicate the endnote to “Romance” too?

With “Dreams”, I assume that’s just an oversight, will correct to meet the scans there.

And yes, I’ll add a production note for sure.

**Comment by robinwhittleton** [comment]
Pushed fixes for those. If you’re worried that there’s more differences to be found then before you continue I’m happy to do another sweep through comparing scans to text, just let me know 🙂

**Comment by vr8hub** [comment]
No, I think you're good. I just wanted to make sure I knew the _intent_ ("match the scans") before I continued. I didn't want to waste your time pointing things you'd already taken into account. (Instead I wanted to waste your time by asking a bunch of questions. lol)
I am continuing on, hope to be finished (my) today. Thank you!
